build/ci.yml

```diff
@@ -10,7 +10,7 @@ pr:
 
 variables:
   Build.Major: 0
-  Build.Minor: 11
+  Build.Minor: 13
```

Comment:
I suggest to set this to 14, since we have some breaking changes coming in with this month.